### DISCUSSION TITLE
why I'm seeing that some people starred a repo that isn't mine and neither I follow that person ?

### DISCUSSION THREAD

**Discussion by abkarim** [community/community#41092] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

![Screenshot](https://user-images.githubusercontent.com/84965867/206454248-2509bb89-5185-43f6-9e12-892d9c0e51aa.png)

At least the feed should describe why am I seeing this.

**Comment by mikekavouras** [comment, marked as answer]
If you hover over the card you should see a `...` in the top right. Click that for additional context/actions. We're working to make this more visible. We're also working on tooling to give you more control over what appears in your feed.

<img width="389" alt="CleanShot 2022-12-09 at 11 51 38@2x" src="https://user-images.githubusercontent.com/627794/206751790-7d540b59-8f4c-4a81-8f08-ac7fb9671544.png">
